test/Microsoft.ML.Tests/Transformers/NormalizerTests.cs

```diff
@@ -324,7 +324,7 @@ public void WhiteningWorkout()
             Done();
         }
 
-        [Fact]
+        [ConditionalFact(typeof(BaseTestBaseline), nameof(BaseTestBaseline.NotFullFramework))]
```

Comment:
we should be investigating why these tests are failing. That is part of why we are doing this exercise.

Reply:
The failure on framework is a native dependency issue, I will try to resolve it before merging this one.

Reply:
i opened the issue https://github.com/dotnet/machinelearning/issues/2104
This is the similar issue i.e while using maml we are not sometimes able to load some modules